src/applications/personalization/profile/components/notification-settings/NotificationRadioButtons.jsx

```diff
@@ -152,52 +157,51 @@ const NotificationRadioButtons = ({
     const buttonAriaDescribedby =
       (checked && ariaDescribedby?.[optionIndex]) || null;
     return (
-      <div
-        key={optionAdditional ? undefined : optionIndex}
-        className="form-radio-buttons"
-      >
-        <div className="errorable-radio-button rb-buttons">
-          <input
-            checked={checked}
-            id={`${id}-${optionIndex}`}
-            name={name}
-            type="radio"
-            onMouseDown={onMouseDown}
-            onKeyDown={onKeyDown}
-            value={optionValue}
-            onChange={handleChange}
-            aria-describedby={buttonAriaDescribedby}
-          />
+      <div key={optionAdditional ? undefined : optionIndex}>

```

Comment:
Would react complain if key is undefined in this instance? I know this isn't really a new change, I'm just curious.

Reply:
Not sure tbh. I'm guessing not since this was copied from the component library radio buttons. And right before I pushed this up I realized optionAdditional doesn't seem to be used in this component at all. So I can likely delete this bit.